nss-day-cohort-57/tabloidmvc-tactful-tortoises
# Ts editcomment

Changes:
CommentController.cs
1.Edit()

CommentRepository.cs
1.UpdateComment()
2.GetCommentById()

To Test:
1. Run Code
2. Select Posts from Nav Bar
3. Select View details of a post
4. Select comments of a post 
5. Select edit comment
6. ensure that comment is edited
